tests/vite-app/README.md

```diff
@@ -0,0 +1,18 @@
+# Tests with Vite and Cypress
+
+This is a Vite project allowing browser-based testing using Cypress.
+
+## How to run the tests
+
+- Clone the repo: `git clone https://github.com/awslabs/aws-jwt-verify`
+- Install dev dependencies and create installable archive: `cd aws-jwt-verify && npm install && npm run pack-for-tests`
+- Install Vite and Cypress dependencies: `cd tests/vite-app && npm install`
+- Start the Vite dev server: `npm run dev`
+- Run the Cypress tests: `npm run cypress:open`
+
+To test the Vite distribution build:
+
+- Start the Vite preview server: `npm run preview`
+- Run the Cypress tests: `npm run cypress:open:preview`
+
+The tokens used in the test were generated using `npm run tokengen`
```

Comment:
This should be a step that's called out for explicitly above? So that you always do this